controls/raddatafilter/datafilter-getting-started.md

```diff
@@ -36,9 +36,7 @@ In order to use __RadDataFilter__ in your project, you need to add references to
 
 After adding references to the aforementioned dlls, you can declare a new __RadDataFilter__ as any normal Silverlight/WPF control.
 
->
-
-To use the __RadDataFilter__ in the XAML you have to add the following namespace declaration:
+> To use the __RadDataFilter__ in the XAML you have to add the following namespace declaration:

```

Comment:
This is seen in many articles but in my opinion it doesn't look good. The note (the blue area) describes that you should see a code snippet. And the code snippet is outside of the note. That's weird. Can you place the text outside of the note? Or add the snippet into the note?